esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -0,0 +1,328 @@
+"""Fixes for ERA5."""
+import datetime
+import logging
+
+import iris
+import numpy as np
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+
+def get_frequency(cube):
+    """Determine time frequency of input cube."""
+    try:
+        time = cube.coord(axis='T')
+    except iris.exceptions.CoordinateNotFoundError:
+        return 'fx'
+
+    time.convert_units('days since 1850-1-1 00:00:00.0')
+    if len(time.points) == 1:
+        if cube.long_name != 'Geopotential':
+            raise ValueError('Unable to infer frequency of cube '
+                             f'with length 1 time dimension: {cube}')
+        return 'fx'
+
+    interval = time.points[1] - time.points[0]
+    if interval - 1 / 24 < 1e-4:
+        return 'hourly'
+
+    return 'monthly'
+
+
+def fix_hourly_time_coordinate(cube):
+    """Shift aggregated variables 30 minutes back in time."""
+    if get_frequency(cube) == 'hourly':
+        time = cube.coord(axis='T')
+        time.points = time.points - 1 / 48
+        time.guess_bounds()
```

Comment:
Yes, this is always the case. However, I think we could remove this line anyway, since they will be added later in the AllVars fix. Agree @bouweandela ?

Reply:
Yes, looks fine to me